web/public/mgmt/config.yml

```
@@ -2522,6 +2522,54 @@ collections:
           - { label: "school/course", value: "school-course" }
           - { label: "object or vehicle", value: "object-vehicle" }
 
+  - name: "raffle-bingo-steps"
+    label: "Raffle Bingo Steps (Navigator with Webflow mappings)"
```

[sand] This should be removed
```suggestion
    label: "Raffle Bingo Steps"
```